src/openforms/js/components/admin/form_design/MissingTranslationsWarning.js

```diff
@@ -82,27 +79,7 @@ MissingTranslationsTable.propTypes = {
   children: PropTypes.arrayOf(PropTypes.object),
 };
 
-const MissingTranslationsWarning = ({form, formSteps}) => {
-  let formStepTranslations = [];
-
-  let formStepsMissingTranslations = [];
-  for (const formStep of formSteps) {
-    if (extractMissingComponentTranslations(formStep.configuration).length) {
-      formStepsMissingTranslations.push(formStep.name);
-    }
-  }
-
-  for (const [index, formStep] of formSteps.entries()) {
-    formStepTranslations = formStepTranslations.concat(
-      extractMissingTranslations(
-        formStep.translations,
-        <FormattedMessage defaultMessage="Steps and fields" description="Form design tab title" />,
-        undefined,
-        ['previousText', 'saveText', 'nextText']
-      )
-    );
-  }
```

Comment:
This one is too much - those are backend translations and not formio component translations. we should be able to keep them.